firmware/src/powhsm/src/bc_err.c

```diff
@@ -74,6 +74,7 @@ void show_error(err_code_t errcode) {
             break;
         }
     }
+    (void)(msg);
```

Comment:
This was only needed when our non-debug build was turning our logs completely off, and thus that variable turned out unused. The `-Wall -Wextra -Werror` compile flags would complain about that and so that fixed it for that particular case. Right now we are not turning our logs off, so will remove it.